llvm/include/llvm/MC/MCELFStreamer.h

```diff
@@ -107,25 +108,51 @@ class MCELFStreamer : public MCObjectStreamer {
     std::string StringValue;
   };
 
+  /// ELF object attributes subsection support
+  struct AttributeSubSection {
+    // [<uint32: subsection-length> NTBS: vendor-name <bytes: vendor-data>]*
+    StringRef Vendor;
+    // <uint8: optional> <uint8: parameter type> <attribute>*
+    unsigned IsMandatory; // SubsectionMandatory::REQUIRED (0),
```

Comment:
You have added enums for these in ARMBuildAttributes.h, should they be used here?

Reply:
Yes, that is a good idea.
Changing the relevant locations.